packages/devextreme/testing/tests/DevExpress.ui.widgets/contextMenu.tests.js

```diff
@@ -273,6 +273,37 @@ QUnit.module('Rendering', moduleConfig, () => {
     });
 });
 
+QUnit.module('Repaint', moduleConfig, () => {
+    QUnit.test('On repaint all submenus should be hidden without console errors (T1257288)', function(assert) {
+        if(!isDeviceDesktop(assert)) {
+            return;
+        }
+
+        const instance = new ContextMenu(this.$element, {
```

Comment:
We usually wrap this kind test in `try {} catch {} finally {}` blocks to catch the issue. This way, it is clear that the error is not expected

```
 try {
            // ...
            $($nestedSubmenuItem).trigger('dxclick');
        } catch(e) {
            assert.ok(false, `error: ${e.message}`);
        } finally {
            assert.ok(true, 'No errors were thrown');
        }
```

Reply:
https://github.com/anna-shakhova/DevExtreme/blob/7b78082bcdfdb698964cbec091c5180e47b996a7/packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/tagBox.tests.js#L7542-L7543